### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
+            </dx:GridViewDataTextColumn>
+            <dx:GridViewDataTextColumn FieldName="ProductName" />
+            <dx:GridViewDataTextColumn FieldName="ProductSales" />
+            <dx:GridViewDataDateColumn FieldName="ShippedDate" />
+        </Columns>
+        <Settings ShowFilterRow="true" />
+    </dx:ASPxGridView>
+    ```
+
+2. Specify a column's [FilterTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewColumn.FilterTemplate) property, add the [ASPxGridLookup](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridLookup) control to the template, populate the editor with columns, and bind it to a data source.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: > Specify a column's [FilterTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewColumn.FilterTemplate) property, add the [ASPxGridLookup](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridLookup) **control** to the template, populate the **editor** with columns, and bind it to a data source.

This is a very good sentence, save for just one thing -- **an implied synonym**. At first, you named `ASPxGridLookup` a control, and then you refer to it as an editor. An alien reader might wonder if "control" and "editor" are the same thing in this context. A possible solution is to name `ASPxGridLookup` editor in both instances.

One more thing -- you encourage your reader to add one editor of the mentioned type to the template -- therefore, the aspect of instance is more important than the aspect of type. That's why I'd like to replace the definite article with indefinite. Add one `ASPxGridLookup` editor to the template. You could add multiple editors to this template, but for the purposes of this example, you need just one.

I hope my explanation is clear enough. Please note that I will not count this as your issue because there is room for different interpretations here.

```suggestion
2. Specify a column's [FilterTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.GridViewColumn.FilterTemplate) property, add an [ASPxGridLookup](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridLookup) editor to the template, populate the editor with columns, and bind it to a data source.
```

Make sure that you have a proper introduction for every entity you use in text.

File: Readme.md [hunk 2]
```diff
@@ -1,22 +1,83 @@
...
-<br/>
+3. Handle the editor's client-side [ValueChanged](https://docs.devexpress.com/AspNet/js-ASPxClientEdit.ValueChanged) event. In the handler, call the grid's client-side `PerformCallback` method to send a callback to the server.
+
+    ```js
+    function Lookup_ValueChanged(s, e) {
+        grid.PerformCallback("FilterByCategories");
+    }
+    ```
+
+4. Handle the grid's server-side `CustomCallback` event. In the handler, create a custom filter criteria based on the edit value and call the grid's [ApplyFilterToColumn](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.ApplyFilterToColumn(DevExpress.Web.GridViewDataColumn-DevExpress.Data.Filtering.CriteriaOperator)) method to apply this filter to the grid.
```
Comment (dmitry-eliseev-devexpress) [inline Readme.md]: Someone would say it's a questionable matter but I can't help but notice this -- "criterion" is a Greek word where "criterion" is singular, and "criteria" is a plural form. Therefore, if you say "criteria" (in plural), you don't need an article.

```suggestion
4. Handle the grid's server-side `CustomCallback` event. In the handler, create custom filter criteria based on the edit value and call the grid's [ApplyFilterToColumn](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView.ApplyFilterToColumn(DevExpress.Web.GridViewDataColumn-DevExpress.Data.Filtering.CriteriaOperator)) method to apply the filter criteria to the grid.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
**Structure and Semantics**

This is a simple step-by-step instruction accompanying a code example. This topic has no structural or semantic issues and serves its purpose. Well done!

**Images**

The only illustration is clear and complies with DevExpress image guidelines. Well done.

**Language**

With a great degree of satisfaction, I can conclude that you nailed down the underlying logic of writing in English. Your language lives up to the highest DevExpress writing standards. But, please take a look at the couple of minor issues I pointed out to avoid them in the future.

**Bottom Line**

I'm pleased with your progress so far but don't rest easy just yet. I expect to see the same level of quality in larger and more complex topics than simple step-by-step readmes.
